command/output.go

```diff
@@ -68,6 +73,20 @@ func (c *OutputCommand) Run(args []string) int {
 		return 1
 	}
 
+        if allOutputs {
```

Comment:
I wonder if the second local var is necessary here - could just check if `name == ""`